packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -32,9 +32,13 @@ const MiniCart = React.forwardRef((props, ref) => {
     // when the MiniCart is open.
     useScrollLock(isOpen);
 
+    const { queries, mutations } = { ...MiniCartOperations };
```

Comment:
```suggestion
```

Reply:
Discussed offline - intentionally not doing the mini cart operations migration because it would require moving _many_ files due to fragment imports.

Reply:
I have changed exported format in the `packages/venia-ui/lib/components/MiniCart/miniCart.gql.js` so it is just `operations` now